src/app/components/loader/loader.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseComponent } from '@components/base-component';
+import { div } from '@components/tags';
+
+import styles from './loader.module.scss';
+
+class LoaderCompoent extends BaseComponent {
+  private spinner = div({});
+
+  constructor() {
+    super({ className: '               grey-modal        '.trim() });
```

Comment:
Using .trim() on a hardcoded string with spaces seems unnecessary. Consider just using 'grey-modal
```suggestion
    super({ className: 'grey-modal' });
```